src/util/error.cpp

```diff
@@ -33,6 +33,8 @@ bilingual_str TransactionErrorString(const TransactionError err)
             return Untranslated("Specified sighash value does not match value stored in PSBT");
         case TransactionError::MAX_FEE_EXCEEDED:
             return Untranslated("Fee exceeds maximum configured by user (e.g. -maxtxfee, maxfeerate)");
+        case TransactionError::MAX_BURN_EXCEEDED:
+            return Untranslated("OP_RETURN output exceeds maximum set by maxburnamount");
```

Comment:
```suggestion
            return Untranslated("OP_RETURN output exceeds maximum configured by user (-maxburnamount)");
```

Reply:
There's no commandline option like for `maxfeerate` so there shouldn't be a hyphen, but I'll change to configured by user, thanks.